### ISSUE TITLE
Replicator panic - AccountNotFound

### ISSUE THREAD

**Issue by HM999** [solana-labs/solana#4984]
Followed all the instructions on a fresh install, see version at end.

It runs for about 30 seconds and can be see on the spy tool.

```
replicating the data with keypair=7DSwipRyxmUReVeFATvbgpXXx1Sya2naGrNUP6fDQKvK gossip_addr=V4(3.9.6.177:0)

thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Custom { kind: Other, error: StringError("AccountNotFound: pubkey=9EbGNwSG6svLze3PpiGrGSzxW8RcvaHFQQSxxb98A9gg: solana client error") }', src/libcore/result.rs:997:5
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
             at src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:39
   1: std::sys_common::backtrace::_print
             at src/libstd/sys_common/backtrace.rs:71
   2: std::panicking::default_hook::{{closure}}
             at src/libstd/sys_common/backtrace.rs:59
             at src/libstd/panicking.rs:197
   3: std::panicking::default_hook
             at src/libstd/panicking.rs:211
   4: std::panicking::rust_panic_with_hook
             at src/libstd/panicking.rs:474
   5: std::panicking::continue_panic_fmt
             at src/libstd/panicking.rs:381
   6: rust_begin_unwind
             at src/libstd/panicking.rs:308
   7: core::panicking::panic_fmt
             at src/libcore/panicking.rs:85
   8: core::result::unwrap_failed
   9: solana::replicator::Replicator::run
  10: solana_replicator::main
  11: std::rt::lang_start::{{closure}}
  12: std::panicking::try::do_call
             at src/libstd/rt.rs:49
             at src/libstd/panicking.rs:293
  13: __rust_maybe_catch_panic
             at src/libpanic_unwind/lib.rs:87
  14: std::rt::lang_start_internal
             at src/libstd/panicking.rs:272
             at src/libstd/panic.rs:388
             at src/libstd/rt.rs:48
  15: main
  16: __libc_start_main
  17: _start
```

```
JSON RPC URL: https://api.testnet.solana.com/
Update manifest pubkey: FnKt2ES9iUJkjoprf2rL62xxBAxZLVgyA4SFexPGotFE
Configuration: /home/ubuntu/.config/solana/install/config.yml
Active release directory: /home/ubuntu/.local/share/solana/install/active_release
Installed version: 
* release date 2019-07-09 13:25:41 +00:00
* download URL https://github.com/solana-labs/solana/releases/download/v0.16.2/solana-release-x86_64-unknown-linux-gnu.tar.bz2

Installation is up to date
```

**Comment by mvines** [comment]
This is probably already fixed by @sagar-solana but adding him to confirm.

**Comment by sagar-solana** [comment]
Yeah I think I took care of this in #4957. No more Rpc unwraps in the replicator.

**Comment by mvines** [comment]
Cool, yeah so @HM999 this fix will be released in the next 0.16 patch release (v0.16.3) later in the week
